Tests/UnitTestsSerialPort/nano.runsettings

```diff
@@ -9,6 +9,6 @@
    </RunConfiguration>
    <nanoFrameworkAdapter>
        <Logging>None</Logging>
-       <IsRealHardware>False</IsRealHardware>
+       <IsRealHardware>True</IsRealHardware>
```

Comment:
This should be False, otherwise this will fail as soon as the unit test are enable in the pipeline build.

Reply:
The fact here is that the unit test will never pass in the CI (if it is working properly), which is why I changed it... The question is, whether we can either check that value, or support more than one file to be run against `remote-host` or `self-host` CI's.

Reply:
Or use `Fakes`

Reply:
Yes, it won't pass because Unit tests that require real hardware can't be run in the pipeline, therefore the unit tests for this library are not enabled. Just like it happens with SPI, for example.

The pattern that we've been following is that the default for this is False.  Unless there a reason for changing it, please let's keep the pattern. No reason for making this one different OK?

Reply:
Ok, I have reverted that change as can be debated later 😉